src/components/global/BaseTag.vue

```diff
@@ -11,13 +11,15 @@ export default defineComponent({
     props: {
         labels: {
             type: Array,
-            required: true,
+            default() {
+                return ['label'];
+            },
```

Comment:
https://vuejs.org/guide/components/props.html#prop-validation

> Object or array defaults must be returned from
a factory function. The function receives the raw
props received by the component as the argument.


e.g.

```
default: () => ['label']
```